README.md

```diff
@@ -1,2 +1,15 @@
 # community-protocols
-Cross-component coordination protocols
+While natively providing the component model and script/style encapsulation that makes web native componentized development possible, these capabilities can also lead to elements that are worlds (possibly "frameworks) unto themselves. While there are use cases where this is _exactly_ the super power that web components intended to bring to the web, there are as many (if not more) where this can cause a headache for web component consumers looking to leverage a selection of components without needing prior knowledge of their internals or the tools that brought them into being. To this end, community protocols for cross-component coordination will expand on the default interoperability of web components all being based at the root on browser APIs by clarifying to component authors paths for communication between elements of dispirate implementation in a way that further empowers component consumers to develop more powerful applications.
```

Comment:
I would prefer to trim this intro down to a pretty perfunctory description of the need to have coordination between components of different origins. A sentence or two should do for now and we can build up more detailed motivation later.

Reply:
Updated version work for ya?